### ISSUE TITLE
xhtml files report trailing spaces not allowed and newline required at end of file

### ISSUE THREAD

**Issue by Standard8** [BenoitZugmeyer/eslint-plugin-html#64]
Using a test xhtml file as follows:

```html
<?xml version="1.0" encoding="UTF-8"?>
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
  <title>Test page</title>
  <script type="application/javascript"><![CDATA[
    document.write("Hello world!");
  ]]></script>
</head>
<body>
</body>
</html>
```

This reports:
```
test.xhtml
   7:1  error  Trailing spaces not allowed.                    no-trailing-spaces (eslint)
  12:1  error  Newline required at end of file but not found.  eol-last (eslint)
```

The only way I can seem to fix it is to remove the indent before the `]]></script>`
